kitchen.yml

```diff
@@ -28,5 +29,5 @@ suites:
    run_list:
    - recipe[test::default]
    attributes:
-      oink_code: <%= ENV['SNORT_LICENCE'] %>
-      community_rules_url: <%= ENV['COMMUNITY_URL'] %>
+      # oink_code: <%= ENV['SNORT_LICENCE'] %>
+      # community_rules_url: <%= ENV['COMMUNITY_URL'] %>
```

Comment:
I put this in so I can pull the OpenSource license code from my env. I need to think how to do this again 🤔

Reply:
circleci context could work?